src/main/java/ru/urururu/sanity/cpp/tools/Clang.java

```diff
@@ -33,4 +25,35 @@ private Clang(String executable) {
 
         return new String[]{executable, filename, "-c", "-emit-llvm", "-femit-all-decls", "-g", "-o", objFile};
     }
+
+    @Override
+    List<String> evaluateVersionIds(String version) {
+        String clangVersion = "clang version";
+        if (version.startsWith(clangVersion)) {
+            version = version.substring(clangVersion.length(), version.indexOf('(')).trim();
+            return createVersionsFamily("clang", version);
+        } else {
+            String appleLlvmVersion = "Apple LLVM version";
+            if (version.startsWith(appleLlvmVersion)) {
+                version = version.substring(appleLlvmVersion.length(), version.indexOf('(')).trim();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Introduce a new variable instead of reusing the parameter "version". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1226)